code/modules/mob/living/carbon/xenomorph/castes/ravager/abilities_ravager.dm

```diff
@@ -157,7 +157,7 @@
 	to_chat(src, "<span class='xenodanger'>Your coursing adrenaline stimulates tissues into a spat of rapid regeneration...</span>")
 	var/current_rage = CLAMP(rage,0,RAVAGER_MAX_RAGE) //lock in the value at the time we use it; min 0, max 50.
 	do_jitter_animation(1000)
-	if(!do_after(src, 50, TRUE, 5, BUSY_ICON_FRIENDLY))

```

Comment:
Is the removal of the icon intended here?

Reply:
there is already the jitteriness acting as indicator, and no visible message.